cmd/main.go

```diff
@@ -32,13 +32,13 @@ import (
 	"sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang"
 	declarativev1 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/declarative/v1"
 	deployimagev1alpha1 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/deploy-image/v1alpha1"
+	multimodulev1alpha1 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/multi-module/v1alpha1"
 	golangv2 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/v2"
```

Comment:
could we add a sample test for this one under the testdata sample?
Also, could we add an e2e test to verify this plugin?

Reply:
Well turns out 14k lines is quite a lot :D